app/src/main/java/org/mozilla/fenix/library/LibrarySiteItemView.kt

```diff
@@ -29,40 +28,45 @@ class LibrarySiteItemView @JvmOverloads constructor(
     defStyleRes: Int = 0
 ) : ConstraintLayout(context, attrs, defStyleAttr, defStyleRes) {
 
-    val titleView: TextView get() = title
+    private val binding = LibrarySiteItemBinding.inflate(
+        LayoutInflater.from(context),
+        this,
+        true
+    )
 
-    val urlView: TextView get() = url
+    val titleView: TextView get() = binding.title
 
-    val iconView: ImageView get() = favicon
+    val urlView: TextView get() = binding.url
 
-    val overflowView: ImageButton get() = overflow_menu
+    val iconView: ImageView get() = binding.favicon
+
+    val overflowView: ImageButton get() = binding.overflowMenu
```

Comment:
nit: Although we're using these properties for easier access there are still calls to `binding.url`, `binding.favicon`, `binding.overflowMenu` below.